lib/mcb/commands/providers/rollover.rb

```diff
@@ -19,7 +19,15 @@
       counts = nil
       bm = Benchmark.measure do
         Provider.connection.transaction do
-          counts = provider.copy_to_recruitment_cycle(new_recruitment_cycle)
+          service = Providers::CopyToRecruitmentCycleService.new(
+            provider: provider,
+            copy_course_to_provider_service: Courses::CopyToProviderService.new(
```

Comment:
Might be worth extracting `Courses::CopyToProviderService` out into a variable, otherwise the nesting makes this a little hard to parse :smile: